src/metabase/models/dashboard.clj

```diff
@@ -97,11 +97,14 @@
 
 (t2/define-before-update :model/Dashboard
   [dashboard]
-  (u/prog1 (maybe-populate-initially-published-at dashboard)
-    (params/assert-valid-parameters dashboard)
-    (parameter-card/upsert-or-delete-from-parameters! "dashboard" (:id dashboard) (:parameters dashboard))
-    (collection/check-collection-namespace Dashboard (:collection_id dashboard)))
-  (maybe-populate-initially-published-at dashboard))
```

Comment:
Why have we stopped populating this timestamp?

Reply:
Was it redundant?

Reply:
it still does, but instead of populating it twice, we can reuse the result when we do it for `u/prog1`

Reply:
Ah, yeah looks like it was done totally redundantly before, unless we were being paranoid about the operations being slow and we wanted to intentionally recalc? Doubt it 🤷